### PR TITLE
[MRG] GTG - New Home v.2

### PR DESCRIPTION
**Pull request by fursinger**
This module and its group module wrapper let you create a 3 column layout. This configuration was requested by the go to market team to better showcase new content presented to our German audience. 

**Important: Clean up the home page that includes dummy cards before merging.**

### CODE REVIEW

**Comment by fursinger** [comment]
For each card, you can define an image or an icon. Both can be omitted if desired. 

**Example for a card with image**
```
{{< card img="api-main" img-width=600 img-alt="" >}}
...
{{< /card >}}
```
`img` is the filename
`img-width` is the natural width of the image
`img-alt` is the alt text of the image

**Example for a card with an icon**
```
{{< card icon="tools.svg" icon-width=40 icon-background="#E5F1FC" >}}
...
{{< /card >}}
```
`icon` is the filename
`icon-width` is the desired with of the icon
`icon-background` is the background-color of the icon stage
The icon will be centered on the icon stage. The icon stage is 92px wide and 92px high.

**Comment by fursinger** [comment]
card-module

File: site/content/german/_index.md [hunk 1]
```diff
@@ -94,51 +94,51 @@ Signieren Sie innert Sekunden rechtsgültig nach EU- und Schweizer Recht, ortsun
...
 ### Schneller abschliessen
 Innert Sekunden verfügen alle Beteiligten über die aktuelle Vertragsversion und können sie mit wenigen Klichs signieren.
 {{% /markdown %}}
 {{< /card >}}
 
-{{< card icon="tools.svg" icon-width=80 icon-background=darkseagreen >}}
+{{< card icon="icons/reduce-cost.svg" icon-width=44 >}}
 {{% markdown %}}
 ### Kosten reduzieren
 Sie senken ihre Ausgaben beim Signieren um bis zu 90% durch effizientere Prozesse und geringere Direktkosten.
```
Comment (Kirastina) [inline site/content/german/_index.md:116]:
```suggestion
Sie senken Ihre Ausgaben beim Signieren um bis zu 90% durch effizientere Prozesse und geringere Direktkosten.
```

**Review by Kirastina — COMMENTED** [review]
Remove typo

**Review by Kirastina — COMMENTED** [review]
Remove typo

**Comment by TillLord** [comment]
I'checked the meta title and meta description of home de/en/fr and some alt texts: Looks good!

**Comment by dkumpf1** [comment]
[DEPLOY-PREVIEW]

**Tested: OK**

I tested this on every browser (Chrome/FF/Chromium) and on mobile (Native/Chrome) with every language (DE/EN/FR) and it looks good. I found some stuff that needs to be improved afterwards together with something in PR 475.

**Comment by betatim** [comment]
This is looking good 😍

While scrolling through the pages I noticed one of the images in the "coming soon" section of https://deploy-preview-463--skribble.netlify.app/features/ is missing

Screenshot 2020-09-30 at 10 07 24

**Comment by michelfernandez** [comment]
> This is looking good 😍
> 
> While scrolling through the pages I noticed one of the images in the "coming soon" section of https://deploy-preview-463--skribble.netlify.app/features/ is missing
> 
> Screenshot 2020-09-30 at 10 07 24

HI Tim, 
Thanks for finding that! We didn't do anything here, but I saw that the tools.svg is in the wrong folder. how can this be? I now put it in the right folder and locally it shows everything correctly. I tried to commit it, but it does not show up!

**Comment by betatim** [comment]
Not sure really why it doesn't show up :-/

On `master` (where it works/the version that is live at the moment) the `tools.svg` is in the `cove/site/static/features/` folder. In this Pull Request it got moved from `cove/site/static/features/` to `cove/site/static/`. I don't know why it got moved, maybe a mistake? So I'd suggest you move it back to the original location and push the commit that does that. Then we see what happens :)

**Comment by michelfernandez** [comment]
> Not sure really why it doesn't show up :-/
> 
> On `master` (where it works/the version that is live at the moment) the `tools.svg` is in the `cove/site/static/features/` folder. In this Pull Request it got moved from `cove/site/static/features/` to `cove/site/static/`. I don't know why it got moved, maybe a mistake? So I'd suggest you move it back to the original location and push the commit that does that. Then we see what happens :)

I could solve it with David. The GitHub Desktop has a problem recently that it suddenly pushes on my repository.

**Comment by dkumpf1** [comment]
[DEPLOY-PREVIEW]

**Tested: OK**

I tested it again and the changes in the footer and the missing Automatic Icon look good from my point of view.

**Pull request merged**